File: javascript/字符串/查找字符串/endsWith.js
// 语法

// string.endsWith(searchvalue, end);

// 参数

// searchvalue 必须 => 需要检索的字符串值
// end 可选 => 可选的整数参数 规定在字符串选中下标前的位置

// 说明

// endsWith() 返回布尔值，表示参数字符串是否在原字符串的尾部。
// 区分大小写,不会改变原有的字符串

let str = "Hello World";

let n1 = str.endsWith('ld');

console.log(n1); // true

let n2 = str.endsWith('H', 1);

console.log(n2);// true

File: python/字符串/查找字符/endswith.py
# 用于判断字符串是否以指定后缀结尾，如果以指定后缀结尾返回 True，否则返回 False。可选参数 "start" 与 "end" 为检索字符串的开始与结束位置。

# str.endswith(suffix[, start[, end]])

# 参数
    # suffix -- 该参数可以是一个字符串或者是一个元素。
    # start -- 字符串中的开始位置。
    # end -- 字符中结束位置。

# 返回值：如果字符串含有指定的后缀返回 True，否则返回 False。

# 示例

str = "Hello World"

print(str.endswith('d')) # Ture

print(str.endswith('e')) # False

print(str.endswith('e', 0, 2)) # True

